src/migrate/Migrator.js

```diff
@@ -32,11 +35,10 @@ function LockError(msg) {
 inherits(LockError, Error);
 
 const CONFIG_DEFAULT = Object.freeze({
-  extension: 'js',
-  loadExtensions: DEFAULT_LOAD_EXTENSIONS,
+  globPatterns: DEFAULT_GLOB_PATTERNS,
```

Comment:
This could also be a non-breaking change. If globPatterns are not specified then the directory is used to build the glob?

Reply:
I thought of that possibility but there is [already a breaking change related to the use case addressed](#2756 ) with this patch. Additionally I thought it was better this way in favor of a more concise API.